src/contact.ts

```diff
@@ -153,6 +160,21 @@ export class Contact implements Sayable {
   }
 
   /**
+   * Check if it's a brand contact
+   *
+   * @returns {boolean|null} True for brand, Flase for contact is not a brand, null for can not get the info
+   *
+   * @example
+   * ```ts
+   * const isBrand = contact.brand()
+   * ```
+   */
+  public brand(): boolean|null {
+    if (!this.obj) return null
```

Comment:
Our framework should always to make sure a `contact` is `ready`, which means `this.obj` should not be empty.

If so, it's more like a bug and should throw an exception.

So my suggestion is to change this function to:

```ts
public brand(): boolean {
  return this.obj && this.obj.brand
}
```